src/agentscope/service/file/check.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Tools for checking files, such as linting and formatting.
+"""
+import subprocess
+
+from agentscope.service.service_response import ServiceResponse
+from agentscope.service.service_status import ServiceExecStatus
+
+
+def exec_py_linting(file_path: str) -> ServiceResponse:
+    """
+    Executes flake8 linting on the given .py file with specified checks and
```

Comment:
About this service function:
- What functionalities are provided in this function is unclear. For example, why we choose F821, F822, F831 ... here? What will this function do exactly?
- Except "F821, F822, F831, E111, E112, E113, E999, E902", do we need to consider the other requirements? 
- It's okay if this function is only added in the example level, but for the AgentScope library it needs to be improved further.

Reply:
New service functions now in example.